### PR TITLE
Stop axis labels duplicating. #315 [Pending squash and cleanup]

### PR DESCRIPTION
**Pull request by noais**

### CODE REVIEW

File: Charts/Classes/Renderers/ChartYAxisRenderer.swift [hunk 1]
```diff
@@ -70,6 +70,9 @@ public class ChartYAxisRenderer: ChartAxisRendererBase
         
         let rawInterval = range / Double(labelCount)
         var interval = ChartUtils.roundToNextSignificant(number: Double(rawInterval))
+        if(!_yAxis.allowRepeatedValues){
```
Comment (danielgindi) [inline Charts/Classes/Renderers/ChartYAxisRenderer.swift]: Please match the code styling of the library

File: .gitignore [hunk 2]
```diff
@@ -28,3 +28,4 @@ DerivedData
 
 Carthage
 Charts.framework.zip
+.DS_Store
```
Comment (danielgindi) [inline .gitignore]: Please remove this

**Comment by danielgindi** [comment]
Friend, all of this should be in a single commit, there's too much noise. Please squash it :-)

**Comment by noais** [comment]
@danielgindi Done

**Comment by liuxuan30** [comment]
I think you can click on "Merge pull request #605 from skywinder/add-change-log-file" from your screen shot. Just create a test project to pratice some. copy-past code sometimes will have unexpected errors.

**Comment by noais** [comment]
The problem was my master wasn't synchronised with the origin.

**Pull request closed without merging**